pkgs/applications/version-management/git-and-tools/glab/default.nix

```
@@ -34,7 +36,7 @@ buildGoModule rec {
   '';
 
   meta = with lib; {
-    description = "An open-source GitLab command line tool";
+    description = "Open-source GitLab command line tool";
```

```suggestion
    description = "Open-source GitLab command line (CLI) tool";
```

For easier searching

---

```suggestion
    description = "GitLab CLI tool bringing GitLab to your command line";
```